src/util/allocate_objects.h

```diff
@@ -95,6 +95,10 @@ class allocate_objectst
     const irep_idt &basename_prefix);
 };
 
+/// Create code allocating object of size `size` and assigning it to `lhs`
+/// \param lhs : pointer which will be allocated
+/// \param size : size of the object
```

Comment:
Nit pick: no need for a space before colon

Reply:
:shell: